src/operations/explore_operation.cpp

```diff
@@ -9,12 +9,20 @@
 #include "util.h"
 
 ExploreOperation::ExploreOperation(const std::vector<geometry_msgs::Point>& path)
-    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 3, 3),
+    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 0.5, 1),
       obstacle_avoidance_path_publisher(node_handle.advertise<ascend_msgs::Path>("/obstacle_avoidance/path", 10)),
       obstacle_avoidance_path_subscriber(
           node_handle.subscribe("/obstacle_avoidance/corrected_path", 10, &ExploreOperation::pathCallback, this)) {}
 
 void ExploreOperation::initialize() {
+    if (path.size() == 0) {
+        point_of_interest = getCurrentPose().pose.position;
+        path.push_back(getCurrentPose().pose.position);
+    }
+
+    point_of_interest = path.back();
+    path.pop_back();
+
     MoveOperation::initialize();
 
     original_path = path;
```

Comment:
If nothing happens during the initialize about the path variable, I would move ` original_path = path;` just above. It looks clearer to me

Reply:
`MoveOperation::initialize()` will alter `path` if the height of the setpoints are too low.